### ISSUE TITLE
How can I plot the validation loss to detect if there is or not over-fit between the training loss?

### ISSUE THREAD

**Issue by DanielYG** [AlexeyAB/darknet#8019]
Hello, 

I can do the training without problems but I can find how I can plot the validation loss (using the validation dataset) during the training to compare it with the training loss. Is it possible? I think it is necessary to determine if there is or not over-fit.

Is it possible to plot in the chart.png the training loss curve and the validation loss curve during the training? 

Thanks!!
